lib/personalized_search/product_preferences_manager.dart

```diff
@@ -103,16 +103,16 @@ class ProductPreferencesManager {
   }
 
   /// Returns whether an attribute is important as per the user preferences.
-  bool isAttributeImportant(String attributeId) {
+  bool? isAttributeImportant(String attributeId) {
     final String importanceId = getImportanceIdForAttributeId(attributeId);
     final int? importanceIndex =
         _availablePreferenceImportances?.getImportanceIndex(importanceId);
-    if (importanceIndex == null ||
-        importanceIndex == PreferenceImportance.INDEX_NOT_IMPORTANT) {
-      // Not important.
-      return false;
+    if (importanceIndex == null) {
+      return null;
     }
-    return true;
+    return importanceIndex == PreferenceImportance.INDEX_NOT_IMPORTANT
```

Comment:
sugg: `importanceIndex != PreferenceImportance.INDEX_NOT_IMPORTANT`

Reply:
I think this is just a matter of taste, I see where you're coming from "returning `true first `false` later" but I'm avoiding negation in the statement

Reply:
I agree double negation doesn't look good